surf/apps/communities/models.py

```diff
@@ -37,6 +41,19 @@ class SurfTeam(UUIDModel):
         related_name='teams',
         blank=True)
 
+    def clean(self):
```

Comment:
We've deleted this model, right? I don't think it's being used any more at least. So adding the check here makes little sense.